src/sql/azdata.proposed.d.ts

```diff
@@ -1264,6 +1264,27 @@ declare module 'azdata' {
 			 * Edges corresponding to the children.
 			 */
 			edges: ExecutionPlanEdge[];
+			/**
+			 * Add badge icon to nodes like warnings and parallelism
```

Comment:
This comment makes it sound like badges are shared across nodes. Could clarify a bit by restating to something like, "Warning/parallelism badges applicable to the current node",